src/components/output/custom-els/PinchZoom/missing-types.d.ts

```diff
@@ -0,0 +1,16 @@
+declare interface CSSStyleDeclaration {
+  willChange: string | null;
+}
+
+// TypeScript, you make me sad.
+// https://github.com/Microsoft/TypeScript/issues/18756
+interface Window {
+  PointerEvent: typeof PointerEvent;
+  Touch: typeof Touch;
+}
+
+declare namespace JSX {
+  interface IntrinsicElements {
+    "pinch-zoom": any
+  }
+}
```

Comment:
This seems to be the right thing to do https://www.typescriptlang.org/docs/handbook/jsx.html#intrinsic-elements